src/gui/creds/oauth.cpp

```diff
@@ -91,10 +101,9 @@ void OAuth::start()
 
                 auto requestBody = new QBuffer;
                 QUrlQuery arguments(QString(
-                    "grant_type=authorization_code&code=%1&redirect_uri=http://localhost:%2")
-                                        .arg(code, QString::number(_server.serverPort())));
+                    "grant_type=authorization_code&code=%1&redirect_uri=http://localhost:%2&code_verifier=%3&scope=openid offline_access")
```

Comment:
Is it safe to construct a query string like that? (yes i see the FullyEncoded below but still) (In code further further below you use proper setQueryItems)

Reply:
Yeah, this is simple enough to be safe.